external/imgui.cmake

```diff
@@ -9,7 +9,7 @@ message (STATUS "Adding imgui ${IMGUI_VERSION} as an external project.")
 
 ExternalProject_Add(imgui
   GIT_REPOSITORY https://github.com/ocornut/imgui.git
-  GIT_TAG ${IMGUI_VERSION}
+  GIT_TAG 848d21b6b56d369fb7976bd88b5f4094ab9ea50a
```

Comment:
Would it be possible to pull from a git tag instead? This has caused problems in the past as the whole commit history had to be pulled to find this exact one.

Reply:
I don't think there is a tag there Unfortunately.
by the way I think I should set the IMGUI_VERSION to the commit hash instead of putting the hash here.